install/kubernetes/cilium/templates/cilium-agent/clusterrolebinding.yaml

```diff
@@ -10,5 +10,5 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: {{ .Values.serviceAccounts.cilium.name | quote }}
-  namespace: {{ .Release.Namespace }}
+  namespace: {{ cilium.namespace }}
```

Comment:
This looks incorrect unfortunately. To use the defined template, you'll have to call include: https://helm.sh/docs/chart_template_guide/named_templates/#the-include-function